### PR TITLE
updated image size and text size

### PR DESCRIPTION
**Pull request by nilsonsasaki**
now persona name and image size has a scaleFactor

### CODE REVIEW

File: lib/ui/vote_persona.dart [hunk 1]
```diff
@@ -44,6 +45,9 @@ class _VotePersonaScreenState extends State<VotePersonaScreen> {
   }
 
   Widget buildPersonaTile(String persona) {
+    final height = MediaQuery.of(context).size.height;
+    final scaleFactor = height/677;
```
Comment (yuumasato) [inline lib/ui/vote_persona.dart:49]: Is it 677 or 667?

**Review by yuumasato — CHANGES_REQUESTED** [review]
On big screens the persona tile looks good, but there are still issues on small screens:
On a Galaxy Nexus, the tiles are overflowing by 1.2 pixels.
Galaxy_Nexus

File: lib/ui/vote_persona.dart (hunk 1 above)
Comment (nilsonsasaki) [reply lib/ui/vote_persona.dart:49]: it is 667

Comment (nilsonsasaki) [reply lib/ui/vote_persona.dart:49]: tested but it won't work as well
maybe we could 
fix it in the next refactor

**Review by yuumasato — APPROVED** [review]

**Pull request merged**